samples/expert_tetris.rb

```diff
@@ -63,9 +100,13 @@ class Tetris
   attr_reader :dt,       # time since the current active piece last dropped a row
               :score,    # the current score
               :lost,     # bool to indicate when the game is lost
-              :pace,     # current game pace - how long until the current piece drops a single row
-              :blocks,   # 2 dimensional array (NX*NY) represeting the tetris court - either empty block or occupied by a piece
-              :actions,  # queue of user inputs collected by the game loop
+              :pace      # current game pace - how long until the current piece drops a single row
+
+  # 2 dimensional array (NX*NY) represeting the tetris court - either empty block or occupied
+  # by a piece
+  attr_reader :blocks
```

Comment:
Would rather we didn't unnecessarily repeat the `attr_reader` just to get the commenting above the items... bet we could just dangle the commas before and allow comments between items generally so the whole section's consistent.